components/Cards/examples/resources/CardsAssets.xcassets/selected.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "selected.png",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
```

Comment:
Looks like the 2x and 3x assets are missing?

Reply:
Do you think these are necessary? We should also consider the increase in the library size.

Reply:
If there is no way to use `MDCIcons`, then these assets are necessary for the examples. Examples resources do not increase the size of the libraries for clients, but do increase the repository size. I think it's a balance between required images to produce the example versus reusing existing images.

Reply:
Done

Reply:
Toggle icons are now using @1, @2 & @3 sizes.